rebar.config

```diff
@@ -20,8 +20,9 @@
         {riak_kv, ".*", {git, "git://github.com/basho/riak_kv.git", {tag, "2.1.7"}}},
         {riak_search, ".*", {git, "git://github.com/basho/riak_search.git", {tag, "2.1.6"}}},
         {riak_control, ".*", {git, "git://github.com/basho/riak_control.git", {tag, "2.1.6"}}},
+        {riak_repl, ".*", {git, "git://github.com/basho/riak_repl.git", {tag, "2.1.8"}}},
```

Comment:
This repository isn't open source yet, thought?  Link is 404.

Reply:
@cmeiklejohn It's about to be!

Reply:
@cmeiklejohn open :)